modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -54,171 +47,180 @@ namespace {
         "Reversed Flow",
         "Toggle to make the flow move in the opposite direction.",
         openspace::properties::Property::Visibility::User
     };
+
     constexpr openspace::properties::Property::PropertyInfo FlowParticleSizeInfo = {
         "ParticleSize",
         "Particle Size",
         "Size of the particles.",
         openspace::properties::Property::Visibility::User
     };
+
     constexpr openspace::properties::Property::PropertyInfo FlowParticleSpacingInfo = {
         "ParticleSpacing",
         "Particle Spacing",
         "Spacing inbetween particles.",
         openspace::properties::Property::Visibility::User
     };
+
     constexpr openspace::properties::Property::PropertyInfo FlowSpeedInfo = {
         "Speed",
         "Speed",
         "Speed of the flow.",
         openspace::properties::Property::Visibility::User
     };
+
     constexpr openspace::properties::Property::PropertyInfo MaskingEnabledInfo = {
         "MaskingEnabled",
-        "Masking",
+        "Masking Enabled",
         "Enable/disable masking. Use masking to show lines where a given quantity is "
         "within a given range, for example, if you only want to see where the "
         "temperature is between 10 and 20 degrees. Also used for masking out line "
         "topologies like solar wind & closed lines.",
-        openspace::properties::Property::Visibility::AdvancedUser
+        openspace::properties::Property::Visibility::User
     };
+
     constexpr openspace::properties::Property::PropertyInfo MaskingMinMaxInfo = {
         "MaskingMinLimit",
         "Lower Limit",
         "Lower and upper limit of the valid masking range.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
+
     constexpr openspace::properties::Property::PropertyInfo MaskingQuantityInfo = {
         "MaskingQuantity",
         "Quantity used for Masking",
         "Quantity used for masking.",
         openspace::properties::Property::Visibility::AdvancedUser
     };
+
     constexpr openspace::properties::Property::PropertyInfo LineWidthInfo = {
         "LineWidth",
         "Line Width",
         "This value specifies the line width of the fieldlines.",
         openspace::properties::Property::Visibility::NoviceUser
     };
+
     constexpr openspace::properties::Property::PropertyInfo TimeJumpButtonInfo = {
         "TimeJumpToStart",
         "Jump to Start Of Sequence",
         "Performs a time jump to the start of the sequence.",
         openspace::properties::Property::Visibility::NoviceUser
     };
 
+    constexpr openspace::properties::Property::PropertyInfo SaveDownloadsOnShutdown = {
+        "SaveDownloadsOnShutdown",
+        "Save Downloads On Shutdown",
+        "This is an option for if dynamically downloaded should be saved between runs "
+        "or not.",
+        openspace::properties::Property::Visibility::User
+    };
+
     struct [[codegen::Dictionary(RenderableFieldlinesSequence)]] Parameters {
```

Comment:
Apologies for asking for more documentation, but this renderable needs a description that will be shown in the docs page. This goes for all the other renderables in this PR as well

Here's an example of what I mean. But maybe merge this PR first and then do another one with just the documentation?   :) 
![image](https://github.com/user-attachments/assets/644f2918-2d1c-4620-ad78-537d5a955b4b)